packages/collector/test/tracing/database/db2/app.js

```diff
@@ -68,7 +68,7 @@ const DB2_TABLE_NAME_3 = process.env.DB2_TABLE_NAME_3 || 'table3';
  */
 let tries = 0;
 const MAX_TRIES = 10;
-const CONNECT_TIMEOUT_IN_MS = 500;
+const CONNECT_TIMEOUT_IN_MS = 2000;
```

Comment:
FYI I increased the timeouts. I am always running into a broken db2 test locally caused by longer connection issues to the db2 server.